spec/controllers/state_file/archived_intake/archived_intake_controller_spec.rb

```diff
@@ -0,0 +1,82 @@
+require 'rails_helper'
+
+describe StateFile::ArchivedIntakes::ArchivedIntakeController, type: :controller do
+  let(:ip_address) { '192.168.0.1' }
+  let(:email_address) { 'test@example.com' }
+  let(:request_instance) { instance_double(StateFileArchivedIntakeRequest) }
+
+  before do
+    allow(controller).to receive(:ip_for_irs).and_return(ip_address)
+    allow(session).to receive(:[]).with(:email_address).and_return(email_address)
```

Comment:
really minor but you can actually just do this:
```suggestion
    session[:email_address] = email_address
```